libraries/standard/http/test/Makefile

```diff
@@ -48,7 +48,7 @@ FUNCTIONS = addHeader \
 include Mock4thewin.mk
 
 # additional dependencies for all tests
-$(TESTS): $(CSDK_ROOT)/platform/posix/clock_posix.o $(CSDK_ROOT)/platform/posix/logging.o
+#$(TESTS): $(CSDK_ROOT)/platform/posix/clock_posix.o $(CSDK_ROOT)/platform/posix/logging.o
```

Comment:
Why is this commented out?

Reply:
As this is not relevant anymore. Will remove the line

Reply:
Removed